runtime/src/main/java/com/xatkit/plugins/moodle/platform/io/MoodleIntentProvider.java

```diff
@@ -33,17 +35,23 @@ public MoodleIntentProvider(MoodlePlatform runtimePlatform, Configuration config
         super(runtimePlatform, configuration);
         this.runtimePlatform.getSocketIOServer().addEventListener(SocketEventTypes.USER_MESSAGE.label, MessageObject.class, 
                 (socketIOClient, messageObject, ackRequest)-> {
-                        Log.info("Received message: {0}", messageObject.getMessage());
-                        Log.info("Received userId: {0}", messageObject.getUserId());
+                        Log.info("Received message: {0}\n", messageObject.getMessage());
+                        Log.info("Received userId: {0}\n", messageObject.getUserId());
+                        Log.info("Received sessionData: {0}\n", messageObject.getCurrentCourseAccess());
                         String userId = messageObject.getUserId();
+                        String currentCourseAccess = messageObject.getCurrentCourseAccess();
```

Comment:
Same as below for the `MessageObject` class, I'd change this to `currentCourseId`